pom.xml

```diff
@@ -16,7 +16,7 @@
     <aspectj.version>1.8.9</aspectj.version>
     <module_name>mod-organizations-storage</module_name>
     <http.port>8081</http.port>
-    <jsonschema_paths>acq-models/mod-orgs/schemas</jsonschema_paths>
+    <jsonschema_paths>acq-models/mod-orgs/schemas,raml-util/schemas</jsonschema_paths>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${basedir}/raml-util</ramlfiles_util_path>
     <raml-module-builder.version>25.0.1</raml-module-builder.version>
```

Comment:
Not related to this PR directly but since you are updating `pom.xml`, you could also update RMB version to `26.2.3`